apps/protocol-frontend/src/components/BulkAttestationModal.tsx

```diff
@@ -13,26 +13,36 @@ import { useUser } from '../contexts/UserContext';
 import { MdCheckCircle } from 'react-icons/all';
 import { AttestationTableType } from '../types/table';
 import pluralize from 'pluralize';
-import useAttestationMint from '../hooks/useAttestationMint';
+import useAttestationBulkMint from '../hooks/useAttestationBulkMint';
 
 interface BulkAttestationModalProps {
   contributions: AttestationTableType[];
 }
 
 const BulkAttestationModal = ({ contributions }: BulkAttestationModalProps) => {
   const { userData } = useUser();
-  const { isLoading: attesting, mutateAsync: mintAttestation } =
-    useAttestationMint();
+  const { isLoading: attesting, mutateAsync: bulkMintAttestation } =
+    useAttestationBulkMint();
   const [currentAttestation] = useState(1);
 
-  const createAttestationsHandler = (contributions: AttestationTableType[]) => {
-    contributions.forEach(async contribution => {
-      try {
-        await mintAttestation(contribution);
-      } catch {
-        return;
-      }
-    });
+  const createAttestationsHandler = async (
+    contributions: AttestationTableType[],
+  ) => {
+    try {
```

Comment:
Should we put this try catch in the hook and remove the handler?

Reply:
query actually never fails, it display the error gracefully. Also, query supposedly have a error handling already. 
It display the `eth_blocknumber` gracefully.

Reply:
It will fail if the rpc is down. I don't think onError catches the error instead it executes when an error occur.